Remotion/Web/Development.WebTesting/Schemas/WebTestingConfiguration.xsd

```diff
@@ -26,6 +26,7 @@
       <xs:attribute name="webApplicationRoot" type="c:RequiredStringType" use="required" />
       <xs:attribute name="screenshotDirectory" type="c:RequiredStringType" />
       <xs:attribute name="logsDirectory" type="c:RequiredStringType" default="." />
+      <xs:attribute name="chromiumDisableSecurityWarningsBehavior" type="c:ChromiumDisableSecurityWarningsBehavior" default="Fail" />

```

Comment:
Despite the default value this configuration should be explicitly added to our integration test configuration files.

Reply:
Fixed in https://github.com/re-motion/Trunk-SVN2Git/pull/9/commits/872c626f5e07b89ed64a04abeda3aa78ab4692d1.